src/main/java/mediathek/update/ProgrammUpdateSuchen.java

```diff
@@ -135,19 +137,13 @@ private void showProgramInformation(boolean showAll) {
      * @param info the remote version number.
      * @return true if there is a newer version
      */
-    private boolean checkForNewerVersion(int info) {
-        boolean result = false;
+    private boolean checkForNewerVersion(Version info) {
 
-        try {
-            final int currentVersion = Integer.parseInt(Const.VERSION);
-            if (info > currentVersion) {
-                result = true;
+            if (Konstanten.MVVERSION.compare(info) == 1) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this if-then-else statement by a single return statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1126)